src/js/grainy.js

```diff
@@ -6,7 +6,7 @@
 
 // permission flags
 
-GRAINY_CONST = {
+var GRAINY_CONST = {
```

Comment:
@jason20c it needs to exist at the global level, so should be `window.GRAINY_CONST`

Perhaps something like `var GRAINY_CONST = window.GRAINY_CONST = {` ?